ui/test/unit/registration/controllers/patientCommonController.spec.js

```diff
@@ -49,6 +51,30 @@ describe('PatientCommonController', function () {
 
     });
 
+    it('checks if the confirmation popup is prompted when the home button is clicked when config is true', function () {
```

Comment:
`'checks that the confirmation popup is prompted when the home button is clicked and the config is enabled'`